products/sle-micro/main.pm

```diff
@@ -9,13 +9,20 @@ BEGIN {
 use utils;
 use testapi;
 use main_common;
+use version_utils ':VERSION';
 
 init_main();
 
 my $distri = testapi::get_required_var('CASEDIR') . '/lib/susedistribution.pm';
 require $distri;
 testapi::set_distribution(susedistribution->new());
 
+if (!is_jeos) {
```

Comment:
why?

Reply:
Maybe this is not strictly needed, but we also do not load jeos needles at all.